testutil/ciao-down/ciao_down.go

```diff
@@ -136,11 +150,14 @@ func prepareFlags(ws *workspace) (*workload, bool, error) {
 	var m mounts
 	var p ports
 	var memGiB, CPUs int
+	var update packageUpgrade
 
 	fs := flag.NewFlagSet("prepare", flag.ExitOnError)
 	vmFlags(fs, &memGiB, &CPUs, &m, &p)
 	fs.BoolVar(&debug, "debug", false, "Enables debug mode")
 	fs.StringVar(&vmType, "vmtype", CIAO, "Type of VM to launch.")
+	fs.Var(&update, "package-upgrade",
+		"Hint to enable or disable update of VM packages.  Should be true or false")
```

Comment:
Because we need tree values:

- ""
- "false"
- "true"

"" means, use the value specified in the workload definition and is the default.